core/src/main/scala/com/salesforce/op/evaluators/OpBinaryClassificationEvaluator.scala

```diff
@@ -122,24 +122,32 @@ private[op] class OpBinaryClassificationEvaluator
     }
   }
 
-  final protected def getBinaryEvaluatorMetric(metricName: ClassificationEvalMetric, dataset: Dataset[_]): Double = {
+  final protected def getBinaryEvaluatorMetric(
+    metricName: ClassificationEvalMetric,
+    dataset: Dataset[_],
+    default: => Double
+  ): Double = {
     val labelColName = getLabelCol
     val dataUse = makeDataToUse(dataset, labelColName)
     new BinaryClassificationEvaluator()
       .setLabelCol(labelColName)
       .setRawPredictionCol(getRawPredictionCol)
       .setMetricName(metricName.sparkEntryName)
-      .evaluate(dataUse)
+      .evaluateOrDefault(dataUse, default = default)
```

Comment:
We should not change the interface - the default has a default value so why change the name??

Reply:
it's an enrichment function for spark `Evaluator`